docs/threat-hunt.md

```diff
@@ -0,0 +1,34 @@
+| [Home](../README.md) |
+|----------------------|
+
+# Threat Hunt
+
+The Threat Hunt module is a place to store and organize your hunts and you can find it under **Threat Intel Management** <picture><source media="(prefers-color-scheme: dark)" srcset="./res/icon-chevron-light.svg"><source media="(prefers-color-scheme: light)" srcset="./res/icon-chevron-dark.svg"><img alt="Fallback image description" src="./res/icon-chevron-dark.svg"></picture> **Hunts**.
+
+Hunts you create here acts as a central repository to associate *Alerts*, *Assets*, *Users*, and other module records with your hunting activity.
+
+1. Click <picture><source media="(prefers-color-scheme: dark)" srcset="./res/icon-add-light.svg"><source media="(prefers-color-scheme: light)" srcset="./res/icon-add-dark.svg"><img alt="Fallback image description" src="./res/icon-add-dark.svg"></picture> **Add**.
```

Comment:
image is not rendering

Reply:
<img width="1254" height="788" alt="image" src="https://github.com/user-attachments/assets/d4691d1f-b309-4f03-bf24-c4e5a14820e9" />